vendor.conf

```diff
@@ -15,7 +15,7 @@ github.com/sirupsen/logrus v1.0.0
 github.com/containers/image 184cc9c74d5b905a74337d24d0648e2214a36dc9
```

Comment:
I see files updated in under vendor for github.com/containers/image but no update for it in vendor.conf.

Reply:
thanks for noticing it.  I forgot to git add the file  :-) fixed now